app/assets/stylesheets/tylium/modules/activities.scss

```diff
@@ -0,0 +1,16 @@
+.activity-filters {
+  input[type="date"] {
+    position: relative;
+  }
+
+  input[type="date"]::-webkit-calendar-picker-indicator {
+    position: absolute;
+    top: 0;
+    right: 0;
+    width: 100%;
+    height: 100%;
+    padding: 0;
+    color: transparent;
+    background: transparent;
+  }
```

Comment:
Isn't this a bit hacky? It doesn't work in Firefox and it's not consistent with other areas of the app where we have a date picker (cards#new for example). Why don't we allow the default behavior and validate the input before submitting?

Reply:
Let's remove it and not cause inconsistencies with other views and cross-browser compatibility issues.